injected/unit-test/messaging.js

```diff
@@ -82,7 +81,7 @@ describe('Android', () => {
             featureName,
             env: 'development'
         })
-        const messaging = new Messaging(messageContextA, config)
+        const messaging = config.intoMessaging(messageContextA)
```

Comment:
By reversing the dependency here, it prevents the Messaging constructor from having to know about every transport